Makefile

```diff
@@ -0,0 +1,31 @@
+SOURCE_DIR=$(PWD)/ChibiOS_18.2.1
```

Comment:
So, you are going for ChibiOS and not for FreeRTOS now?

Reply:
Yes, we decided to go with ChibiOS as it has HAL preconfigured. We would have to configure that manually in FreeRTOS

Reply:
The ChibiOS version that I used four years ago was of course an older one; however, I hope that you could use some of the code snippets from there?

Reply:
Most likely, yes. We will work on merging the projects today.